.github/workflows/stale_cleaner.yml

```
@@ -0,0 +1,37 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+          close-issue-message: 'This issue has been closed due to inactivity. If you believe this issue is still relevant, please feel free to reopen it with additional context or information.'
+          stale-pr-message: 'This PR is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+          close-pr-message: 'This PR has been closed due to inactivity. If you believe this PR is still relevant, please feel free to reopen it with additional context or information.'
+          delete-branch: true
+          days-before-stale: 30
+          days-before-close: 5
+          any-of-issue-labels: 'bug'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)